src/components/app/StudentRoute.tsx

```diff
@@ -23,7 +23,7 @@ const StudentRoute: React.FC<StudentRouteProps> = ({ component: Component, user,
       rest.getUser();
       return <div>...Getting User...</div>
     }
-    if (user.type === UserType.Student || user.type === UserType.Admin) {
+    if (user.type === UserType.Student || user.type === UserType.Admin || user.type === UserType.Builder || user.type === UserType.Editor) {
```

Comment:
I am still not sure about this. I really want only student role to have access to play and then make the the play preview route separate.